aloscene/tensors/spatial_augmented_tensor.py

```diff
@@ -12,21 +12,48 @@
 
 
 class SpatialAugmentedTensor(AugmentedTensor):
+    """Spatial Augmented Tensor. Used to represets any 2D data. The spatial augmented tensor can be used as a
+    basis for images, depth or and spatially related data. Moreover, for stereo setup, the augmented tensor
+    can encode data about the baseline and/or the side of the current plane ("left" or "right").
+
+    cam_intrinsic: CameraIntrinsic
+        Camera Intrinsic. If provided, the focal_length, plane_size, principal_point, camera_side & baseline will not
+        be used. An error will be raised if theses parameters are provided since using the `cam_intrinsic` and theses
+        parameters could be ambigious.
+    cam_extrinsic: CameraExtrinsic
+        Camera extrinsic parameters as an homogenious transformation matrix.
+    camera_side : {'left', 'right', None}
+        If part of a stereo setup, will encode the side of the camere "left" or "right" or None.
+    baseline: float | None
```

Comment:
Why not just only in disparity ?
It's very specific compared to SpatialAugmentedTensor which represent only grid based data

Reply:
An alternative would be to keep track of the "sensor" which as produce the information, namely the frame?
But I agree it's a simpler this way.